packages/core/contracts/insured-bridge/BridgeAdmin.sol

```diff
@@ -116,6 +116,19 @@ contract BridgeAdmin is BridgeAdminInterface, Ownable, Lockable {
         emit SetDepositContracts(_chainId, _depositContract, _messengerContract);
     }
 
+    /**
+     * @notice Enables the current owner to transfer ownership of a set of owned bridge pools to a new owner.
+     * @dev Only callable by the current owner.
+     * @param _bridgePools array of bridge pools to transfer ownership.
+     * @param _newAdmin new admin contract to set ownership to.
+     */
+    function transferBridgePoolAdmin(address[] memory _bridgePools, address _newAdmin) public onlyOwner nonReentrant() {
+        for (uint16 i = 0; i < _bridgePools.length; i++) {
```

Comment:
Oh I didn't see that you pass `_bridgePools` in as a function param, I think its OK to revert back to uint8. Your call, sorry